app/frontend/src/registry/components/people/Notes.vue

```diff
@@ -50,16 +55,88 @@
               </b-btn>
             </div>
           </b-modal>
+          <b-modal
+              v-model="confirmDeleteModal"
+              centered
+              title="Confirm Deletion"
+              @shown="focusDeleteModal"
+              :return-focus="$refs.noteInputCancelBtn">
+              <p>Are you sure you want to delete this note?</p>
+              <div v-if="activeNote" >
+                <p class="font-weight-bold">"{{activeNote.note}}"</p>
+              </div>
+            <div slot="modal-footer" class="buttons">
+              <b-btn
+                variant="light"
+                @click="confirmDeleteModal=false"
+                ref="cancelDeleteBtn"
+              >
+                Cancel
+              </b-btn>
+              <b-btn
+                variant="danger"
+                @click="confirmDeleteModal=false;deleteNote()"
+              >
+                Delete
+              </b-btn>
+            </div>
+          </b-modal>
+          <b-modal
+              v-model="confirmEditNoteModal"
+              centered
+              title="Editing Note"
+              @shown="focusEditNoteModal"
+              :return-focus="$refs.noteInputCancelBtn"
+            >
+            <div>
+              <b-form-textarea
+                id="editNoteTextArea"
+                v-model="noteContentEdit"
+                placeholder="Edit Note..."
+                rows="4"
+                max-rows="6"
+              />
+            </div>
+            <div slot="modal-footer" class="buttons">
+              <b-btn variant="light" @click="confirmEditNoteModal=false" ref="cancelEditNoteCancelBtn">
+                Cancel
+              </b-btn>
+              <b-btn variant="primary" @click="notePatchHandle()">
+                Submit
+              </b-btn>
+            </div>
+          </b-modal>
         </b-form>
       </div>
       <div id="notesList" ref="notes">
         <div class="mt-5" v-if="!notes || !notes.length">
           <b-row><b-col>No notes for this record.</b-col></b-row>
         </div>
-        <div class="mt-5" v-if="notes && notes.length">
-          <div class="mt-3" v-for="(note, index) in notes" :key="`note ${index}`" :id="`note-${index}`">
+        <div class="mt-5 note-container" v-if="notes && notes.length">
+          <div class="note" v-for="(note, index) in notes" :key="`note ${index}`" :id="`note-${index}`">
+            <p>
               <span class="font-weight-bold">{{ note.author }}</span> ({{ note.date | moment("MMMM Do YYYY [at] LT") }}):
               {{ note.note }}
```

Comment:
Might be a good idea to wrap this in a `<span>` and give it a `word-break: break-all;` styling in case extended text is added to the note. Probably only for instances when the note is a link.
<img width="1049" alt="Screenshot 2024-03-28 at 12 34 44 PM" src="https://github.com/bcgov/gwells/assets/62899351/ff815850-b92d-4b1b-9303-b6038aac6f26">

Reply:
![image](https://github.com/bcgov/gwells/assets/62873746/edd5089a-df7f-426f-91d8-6c6c8a003563)

Implemented
Definitely posted these on the opposite comments